__tests__/articles-article_id.test.js

```diff
@@ -0,0 +1,56 @@
+const request = require("supertest");
+const app = require("../index");
+const db = require("../db/connection");
+const seed = require("../db/seeds/seed");
+const data = require("../db/data/test-data");
+const jestSorted = require("jest-sorted");
+
+afterAll(() => {
+  db.end();
+});
+
+beforeEach(() => {
+  return seed(data);
+});
+
+describe("GET /api/articles/:article_id", () => {
+  it("responds with an array of article objects", () => {
+    return request(app)
+      .get("/api/articles/3")
+      .then((response) => {
+        expect(response.status).toBe(200);
+        expect(response.body).toBeInstanceOf(Object);
+        expect(response.body).toHaveProperty("article");
+        expect(response.body.article).toEqual({"article_id": 3, "article_img_url": "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700", "author": "icellusedkars", "body": "some gifs", "comment_count": "2", "created_at": "2020-11-03T09:12:00.000Z", "title": "Eight pug gifs that remind me of mitch", "topic": "mitch", "votes": 0});
```

Comment:
This test checks the properties and therefore the following assertions are not needed.